### PR TITLE
Archive and trash management

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
- Users can add notes to trash or add to archives.
- user can also unarchive or restore the notes from trash

### CODE REVIEW

**Review by shashikantmaurya0007 — APPROVED** [review]
code looks good to be merged

**Review by anjali1102 — APPROVED** [review]

File: src/Utils/services.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+export const deletefromAllNotes = async (note, token) => {
+    const { _id } = note;
+    try {
+        await axios.delete(`/api/notes/archives/${_id}`,{headers:{authorization: token}});
+    } catch (err) {
+        console.log(err)
+  }
+}
+
+// -----------------------------------------------------------------------------------------------------------------------------
```
Comment (ashishkatiyar2404) [inline src/Utils/services.js]: remove these lines and write down comments on what the below function returns or does.

File: src/Utils/services.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+export const deletefromArchived = async (note, token) => {
+  console.log(token)
+    try {
+        await axios.delete(`/api/archives/delete/${note._id}`,{headers:{authorization: token}});
+    } catch (err) {
+        console.log(err)
+  }
+}
+
+//-------------------------------------------------------------------------------------------------------------------------------
```
Comment (ashishkatiyar2404) [inline src/Utils/services.js]: same here

**Review by ashishkatiyar2404 — APPROVED** [review]
code looks good, well done Maruthi pls look into suggestion and you can merge.

**Pull request merged**